cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/loadbalancer/AwsTargetGroupMetadata.java

```diff
@@ -0,0 +1,62 @@
+package com.sequenceiq.cloudbreak.cloud.model.loadbalancer;
+
+import java.util.Objects;
+
+/**
+ * Contains information about listeners and target groups attached to a load balancer. In AWS each listener
+ * listens on a single port, and forwards traffic received on that port to a target group, which distributes
+ * the traffic among the instances in the target group on a defined traffic port. Listeners and target groups
+ * each have their own ARN. In the cloudbreak implementation, the listener port and the target group traffic
+ * port are the same, so they are represented by a single 'port' member.
+ */
+public class AwsTargetGroupMetadata {
```

Comment:
I would use the same approach here, not to store AWS related stuff in cloud model directly.

Reply:
This class doesn't exist anymore.